torch/_dynamo/variables/builtin.py

```diff
@@ -1018,6 +1034,11 @@ def call_sum(self, tx, seq, **kwargs):
                 {},
             )
 
+    def call_StopIteration(self, tx):
+        from .user_defined import UserDefinedObjectVariable
+
+        return UserDefinedObjectVariable(StopIteration())
```

Comment:
It's kind of weird to make this a UDOV, since this is a definitely a built-in

Reply:
Well, it is a builtin function, but every instance of StopIteration() is technically a unique object